.github/workflows/continuous-integration.yml

```diff
@@ -43,7 +43,7 @@ jobs:
           - dependencies: "php-http/guzzle7-adapter"
             php-version: "8.2"
             symfony-deprecations-helper: "weak"
-          - dependencies: "php-http/guzzle7-adapter"
+          - dependencies: "php-http/guzzle7-adapter php-http/throttle-plugin"
```

Comment:
once #463 is merged, move this to the require-dev section and adjust the test to always include throttle plugin configuration